api/fixtures/example.go

```diff
@@ -512,6 +513,12 @@ func (o ExampleOptions) Resources() *ExampleResources {
 		cluster.Spec.PausedUntil = &o.PausedUntil
 	}
 
+	if o.OperatorHub != nil {
+		cluster.Spec.Configuration = &hyperv1.ClusterConfiguration{
```

Comment:
In case we add something else that also modifies configuration, I would add a check like:

```
if cluster.Spec.Configuration == nil {
  cluster.Spec.Configuration = &hyperv1.Configuration{}
}
cluster.Spec.Configuration.OperatorHub = o.OperatorHub
```